pkg/dataset/config.go

```diff
@@ -38,16 +44,17 @@ type Config struct {
 
 // Metrics contains dataset configuration options for metrics data.
 type Metrics struct {
-	ChunkInterval   DayDuration `yaml:"default_chunk_interval"`
-	Compression     *bool       `yaml:"compress_data"` // Using pointer to check if the the value was set.
-	HALeaseRefresh  DayDuration `yaml:"ha_lease_refresh"`
-	HALeaseTimeout  DayDuration `yaml:"ha_lease_timeout"`
-	RetentionPeriod DayDuration `yaml:"default_retention_period"`
+	ChunkInterval   util.DayDuration `yaml:"default_chunk_interval"`
+	Compression     *bool            `yaml:"compress_data"` // Using pointer to check if the value was set.
+	HALeaseRefresh  util.DayDuration `yaml:"ha_lease_refresh"`
+	HALeaseTimeout  util.DayDuration `yaml:"ha_lease_timeout"`
+	RetentionPeriod util.DayDuration `yaml:"default_retention_period"`
+	Downsample      `yaml:"downsample,omitempty"`
```

Comment:
Why don't we add a field here? It will add more structure thus easier to work with `Metrics` struct